CMakeLists.txt

```diff
@@ -20,29 +23,81 @@ add_compile_definitions(TFLITE_BUILD_WITH_XNNPACK_DELEGATE)
 
 # This assumes that tensorflow got checked out as submodule.
 # True, if `git clone` had the additional option `--recursive`.
-add_subdirectory(tensorflow/tensorflow/lite 
+add_subdirectory(tensorflow/tensorflow/lite
   "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL)
 
 add_compile_definitions(DEEPSEG_VERSION=${DEEPSEG_VERSION})
 
+add_library(backscrub
+  libbackscrub.cc
+  transpose_conv_bias.cc)
+
+target_link_libraries(backscrub
+  tensorflow-lite ${CMAKE_DL_LIBS}
+  opencv_core
+  opencv_imgproc
+)
+
 add_executable(deepseg
   deepseg.cc
-  loopback.cc 
+  loopback.cc
```

Comment:
Maybe also put the handling of the loopback device into the library. Can be a separate set of API calls from the user side, but given the minimal interface required (`bs_vcam_open`, `bs_vcam_openfd`, `bs_vcam_frame_push`, `bs_vcam_close`, plus maybe some helper functions) this might be worth the split. Or have the vcam stuff split into its own library too.

Reply:
I'll go with a separate library, given the initial use case of incorporating backscrub into OBS Studio which will do all the the I/O, and provides a cut point to port to other operating systems that have other video loopback APIs.